CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.9.0] - 2016-12-21
```

Comment:
@asadiqbal08 https://github.com/edx/edx-platform/pull/14203 is the corresponding PR.